mixins/powerState/setDevicePowerStateMixin.js

```diff
@@ -56,6 +56,12 @@ const setDevicePowerState = {
       params.switch = stateToSwitch;
     }
 
+    if (state === 'custom') {
+      delete params.switch;
+    }
+
+    Object.assign(params, extraParams);
```

Comment:
I could also move this into the `state === 'custom'` if-condition to make sure this is only applied with `custom` state

Reply:
done